app/models/mixins/relationship_mixin.rb

```diff
@@ -221,27 +221,24 @@ def ancestors_count(*args)
   def path_rels(*args)
     options = args.extract_options!
     rel = relationship(:raise_on_multiple => true) # TODO: Handle multiple nodes with a way to detect which node you want
-    rels = rel.nil? ? [] : rel.path
+    rels = rel.nil? ? [Relationship.new(:resource => self)] : rel.path
```

Comment:
I'm thinking this pattern of `Relationship.new(:resource => self)` should move into a new method to describe intent.  Perhaps new_relationship to mirror the naming of init_relationship (and then have the code live next to init_relationship to show their commonality)?  Or perhaps `relationship_for_isolated_root`?  Not sure...naming is hard :)